jobs/build/update-microshift-pocket/Jenkinsfile

```diff
@@ -63,7 +63,7 @@ node {
         """
     )
 
-    buildlib.doozer("--working-dir ${DOOZER_WORKING} --assembly ${params.ASSEMBLY} --group openshift-${version} config:plashet --base-dir ${ORG_PLASHET_DIR} --name repos --repo-subdir os -i microshift --arch x86_64 unsigned --arch s390x unsigned --arch ppc64le unsigned --arch aarch64 unsigned from-tags -t rhaos-${version}-rhel-${params.RHEL_TARGET}-candidate NOT_APPLICABLE")
+    buildlib.doozer("--working-dir ${DOOZER_WORKING} --group openshift-${version} -r microshift config:plashet --base-dir ${ORG_PLASHET_DIR} --name repos --repo-subdir os -i microshift --arch x86_64 unsigned --arch s390x unsigned --arch ppc64le unsigned --arch aarch64 unsigned from-tags -t rhaos-${version}-rhel-${params.RHEL_TARGET}-candidate NOT_APPLICABLE")
```

Comment:
Could you make it `--rpms` rather than `-r`? For scripts I think long form options are better for readability.

Reply:
Is there a reason to remove the `--assembly` option? Doozer `from-tags` command [seems to be using it](https://github.com/openshift/doozer/blob/master/doozerlib/cli/config_plashet.py#L488)

Reply:
@joepvd makes sense.
@locriandev `buildlib.doozer` already adds `--assembly` option.